doc/classes/Timer.xml

```diff
@@ -15,20 +15,21 @@
 			<return type="float">
 			</return>
 			<description>
-				Return the time left for timeout in seconds if the timer is active, 0 otherwise.
+				Returns the time left for timeout in seconds if the timer is active, 0 otherwise.
```

Comment:
"time left for timeout" sounds a bit awkward:

Returns the timer's remaining time in seconds. Returns 0 if the timer is inactive.